ydb/docs/en/core/reference/docker/environment.md

```diff
@@ -0,0 +1,31 @@
+| [`YDB_ADDITIONAL_LOG_CONFIGS`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L48)            | `string`   | `None`                 |                              | Sets additional logging levels in format: `level`:`level value`. If you need to enter multiple levels, then enter them separated by commas.                                            |
+| [`YDB_FEATURE_FLAGS`](https://GitHub.com/ydb-platform/ydb/blob/69a57074e4c259aea0bbb9a735c5ed821743629c/ydb/public/tools/lib/cmds/__init__.py#L395) | `string` | `None` | | Comma-separated list of [experimental features](https://GitHub.com/ydb-platform/ydb/blob/69a57074e4c259aea0bbb9a735c5ed821743629c/ydb/core/protos/feature_flags.proto) YDB |
+| [`YDB_ENABLE_COLUMN_TABLES`](https://GitHub.com/ydb-platform/ydb/blob/69a57074e4c259aea0bbb9a735c5ed821743629c/ydb/tests/library/harness/kikimr_config.py#L86) | `0` or `1` | `0` | | Enables column tables in YDB|
+<!---
+| [`YDB_PQ_CLIENT_SERVICE_TYPES`](https://GitHub.com/ydb-platform/ydb/blob/8fefc809c83829d8d8b886e82534d009de4c8826/ydb/public/tools/lib/cmds/__init__.py#L297)               | `string`   | `None`                 |                              | Client type `persistent queue`.                                                                                                                                                        |
+| [`YDB_PQ_TOPICS_ARE_FIRST_CLASS_CITIZEN`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L77) |  `0` or `1` | `1` | |  required for logbroker federation, not used in normal ydb . |
+| [`YDB_PQ_CLUSTER_TABLE_PATH`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L78)             | `string` | `None`  | |  required for logbroker federation, not used in normal ydb . |
+| [`YDB_PQ_VERSION_TABLE_PATH`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L79)             | `string` | `None`  | |  required for logbroker federation, not used in normal ydb . |
+| [`YDB_PQ_ROOT`](https://GitHub.com/ydb-platform/ydb/blob/b1c590828b222c839dedecd8e6e79413ef5b7eec/ydb/tests/library/harness/kikimr_config.py#L80)                           | `string` |  `None` |  | required for logbroker federation, not used in normal ydb.         |
+| [`YDB_ENABLE_PQCD`](https://GitHub.com/ydb-platform/ydb/blob/8fefc809c83829d8d8b886e82534d009de4c8826/ydb/public/tools/lib/cmds/__init__.py#L310)                           | `0` or `1` | `0`                    |                              | Enable usage `persistent queue cluster discovery`                                                                                                                                      |
+-->
```

Comment:
Why is this commented out?

Reply:
This envs also used in docker image local-ydb, but we dont want to show it - bc it is internal

Reply:
removed all of them